modules/tensor_mechanics/src/materials/cohesive_zone_model/BiLinearMixedMode.C

```diff
@@ -0,0 +1,203 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "BiLinearMixedMode.h"
+#include "libmesh/utility.h"
+
+registerMooseObject("TensorMechanicsApp", BiLinearMixedMode);
+
+InputParameters
+BiLinearMixedMode::validParams()
+{
+  InputParameters params = CZMComputeLocalTractionTotalBase::validParams();
+  params.addRequiredParam<Real>("stiffness", "Penalty stiffness.");
```

Comment:
if this is the interpenetration penalty stiffness call the parameter and the variable `penalty_stiffness`

Reply:
done.